src/pykeen/models/unimodal/tucker.py

```diff
@@ -76,11 +76,15 @@ class TuckER(EntityRelationEmbeddingModel):
     #: The default parameters for the default loss function class
     loss_default_kwargs = {}
 
+    def _reset_parameters_(self):  # noqa: D102
```

Comment:
why did these lines change?

Reply:
Good question :D 
Fixed in ee30b4257a715ae5838e24651241b353887e0ceb